### PR TITLE
mode 128 is 320x240

### PR DESCRIPTION
**Pull request by irmen**
see https://cx16forum.com/forum/viewtopic.php?p=34939#p34939

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -708,7 +708,7 @@ This is the first release of x16-rom by the X16Community team
 ### Release 35
 
 * API Fetures
-	* new KERNAL API: low-level and high-level 320x200@256c bitmap graphics
+	* new KERNAL API: low-level and high-level 320x240@256c bitmap graphics
```
Comment (mooinglemur) [inline README.md:711]: I'm very reluctant to change the content of old changelogs.

File: README.md [hunk 2]
```diff
@@ -773,7 +773,7 @@ This is the first release of x16-rom by the X16Community team
 	* complete jump table bridge
 * KERNAL: memory size detection
 * KERNAL: faster IRQ entry
-* GEOS: converted graphics library to VERA 320x200@256c
+* GEOS: converted graphics library to VERA 320x240@256c
```
Comment (mooinglemur) [inline README.md:776]: I'm very reluctant to change the content of old changelogs.

File: inc/io.inc [hunk 3]
```diff
@@ -92,7 +92,7 @@ d2ora	=via2+15
 ;
 ; VERA Video RAM Layout
 ;
-; $0:0000 - $1:2BFF  320x240@256c Bitmap [320x200: - $0:F9FF]
+; $0:0000 - $1:2BFF  320x240@256c Bitmap
```
Comment (mooinglemur) [inline inc/io.inc:95]: This one we should change though.

**Review by mooinglemur — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (irmen) [reply README.md:711]: i'll discard this change

**Review by mooinglemur — APPROVED** [review]

**Comment by mooinglemur** [comment]
thanks!

**Pull request merged**